### DISCUSSION TITLE
Penempatan kode iklan pada diantara postingan di halaman index?

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#197] [Q&A]
Kang, untuk penempatan iklan diantara postingan di halaman index bagaimana ya kang? kita ingin menampilkan iklan, setiap 3 atau 4 atau 5 postingan akan tampil 1 iklan.

settingan yang mirip blogger gitu. karena kalau ngandelin iklan otomatis, muncul iklan nya terlalu random, kadang suka nongol di tempat yg tidak diinginkan

**Comment by taufik-nurrohman** [comment]
Belum ada solusi mas, soalnya tiap layout punya struktur HTML yang beda-beda. Lain lagi kasusnya kalau mau meletakkan iklan di dalam posting, cenderung lebih mudah karena konten posting paling isinya cuma paragraf.
